src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/crud-table/crud-table.component.ts

```diff
@@ -131,7 +142,27 @@ export class CRUDTableComponent implements OnInit {
   updateSelection(selection: CdTableSelection) {
     this.selection = selection;
   }
+
   setExpandedRow(event: any) {
     this.expandedRow = event;
   }
+
+  authExport() {
+    let entities: string[] = [];
+    this.selection.selected.forEach((row) => entities.push(row.entity));
+    this.cephUserService.export(entities).subscribe((data: string) => {
+      const modalVariables = {
+        titleText: $localize`Ceph user export data`,
+        buttonText: $localize`Finish`,
```

Comment:
If this is only applicable for `authExport` probably this should be put under `cluster/auth-export.component`?

Reply:
I didn't want to create a new file for singular function for now. We can cut-paste when this gets longer.